packages/dev/core/src/Meshes/meshUVSpaceRenderer.ts

```diff
@@ -191,6 +272,70 @@ export class MeshUVSpaceRenderer {
         texture.setMaterialForRendering(this._mesh, MeshUVSpaceRenderer._GetShader(this._scene));
 
         this.texture = texture;
+

```

Comment:
You must also set `this._userCreatedTextureConfigured = true;`, otherwise `_configureUserCreatedRTT` will also be called when the texture is created internally (and you'll get WebGL errors in the console).

To be clearer, I'd rename `_userCreatedTextureConfigured` to `_configureUserCreatedTexture` and reverse the operation: I'd initialize it to `true` and set it to `false` in `_configureUserCreatedRTT` and `_createDiffuseRTT`.